activation/activation_test.go

```diff
@@ -481,414 +347,313 @@ func TestBuilder_RestartSmeshing(t *testing.T) {
 		for worker := 0; worker < 10; worker += 1 {
 			eg.Go(func() error {
 				for i := 0; i < 100; i++ {
-					builder.StartSmeshing(types.Address{}, PostSetupOpts{})
-					builder.StopSmeshing(true)
+					_ = builder.StartSmeshing(types.Address{}, PostSetupOpts{})
+					_ = builder.StopSmeshing(true)
 				}
 				return nil
 			})
 		}
-		eg.Wait()
+		require.NoError(t, eg.Wait())
 	})
 }
 
 func TestBuilder_StopSmeshing_failsWhenNotStarted(t *testing.T) {
-	cdb := newCachedDB(t)
-	atxHdlr := newAtxHandler(t, cdb)
-	builder := newBuilder(t, cdb, atxHdlr)
-
-	require.ErrorContains(t, builder.StopSmeshing(true), "not started")
+	tab := newTestBuilder(t)
```

Comment:
I'm curious - why `tab` and not `tb`? **T**est **B**uilder 😃

Reply:
tb, as a variable, is already used in some methods for testing.TB.